mmaction/core/bbox/transforms.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+import torch
+
+
+def bbox_overlaps(bboxes1, bboxes2, mode='iou', aligned=False):
+    """Calculate overlap between two set of bboxes.
+
+    Args:
+        bboxes1 (Tensor): shape (m, 4)
+        bboxes2 (Tensor): shape (n, 4), if 'aligned' is True, then m and n
+            must be equal.
+        mode (str): "iou" (intersection over union) or iof (intersection over
+            foreground).
+        aligned (bool): If 'aligned' is False, then calculate the ious between
+            each bbox of bboxes1 and bboxes2, otherwise the ious between each
+            aligned pair of bboxes1 and bboxes2.
+
+    Returns:
+        ious(Tensor): shape (m, n) if aligned == False else shape (m, 1)
+    """
+
+    assert mode in ['iou', 'iof']
+
+    rows = bboxes1.size(0)
+    cols = bboxes2.size(0)
+    if aligned:
+        assert rows == cols
+
+    if rows * cols == 0:
+        return bboxes1.new(rows, 1) if aligned else bboxes1.new(rows, cols)
+
+    if aligned:
+        lt = torch.max(bboxes1[:, :2], bboxes2[:, :2])  # [rows, 2]
+        rb = torch.min(bboxes1[:, 2:], bboxes2[:, 2:])  # [rows, 2]
```

Comment:
not good names for `lt` and `rb`

Reply:
OK, the full name will be used instead of the abbreviation.